src/status_im/events.cljs

```diff
@@ -2149,3 +2147,45 @@
  :shake-event
  (fn [cofx _]
    (logging/show-logs-dialog cofx)))
+
+;; ENS
+
+(handlers/register-handler-fx
```

Comment:
```
(fx/defn set-state
  {:events [set-username-candidate]}
  [{:keys [db]} custom-domain? username]
 ...)
```

Reply:
there is no need to declare this event here

Reply:
:+1: